buildroot-external/genimage/hdimage-mbr.cfg

```diff
@@ -0,0 +1,6 @@
+hdimage {
+	partition-table-type = "mbr"
+	align = 1M
+	disk-signature = 0x48617373
```

Comment:
I guess to make it binary same as we have today this makes sense. But maybe it would be better to be random? :thinking:

Reply:
Not sure, we'll have to find out if we can use a different approach from hardcoded partition ID in `mbr-part.rules` and U-Boot scripts (just grep this number in the repo for reference).

Reply:
Ah I see. Ok, yeah then we definitely need to keep it as-is for now!